ts/src/base/Exchange.ts

```diff
@@ -3418,6 +3420,18 @@ export default class Exchange {
                     }
                 }
                 return markets[0];
+            } else {
+                // temporary support for old futures
+                if (symbol.indexOf ('-') >= 0) {
+                    for (let i = 0; i < this.futureTypes.length; i++) {
```

Comment:
so, if this is `future` symbol, we replace
```
BTC/USDT:BTC-230611
```
with each possible symbol, i.e.
```
BTC/USDT:BTC-Q230611
```
and search if that symbol exists.